src/mono/mono/mini/CMakeLists.txt

```
@@ -318,6 +318,9 @@ endif()
 if(HOST_WIN32)
 set(mini_sources "${mini_sources};${PROJECT_BINARY_DIR}/../../NativeVersion.rc") # this is generated by GenerateMonoVersionFile in mono.proj
 endif()
+if(HOST_LINUX)
```

I think we can do this on other non-Windows platforms (i.e. macOS) too:

```suggestion
else()
```

---

OK, I'll change that.

---

Done